src/vs/base/browser/ui/dropdown/dropdownActionViewItem.ts

```diff
@@ -134,6 +139,26 @@ export class DropdownMenuActionViewItem extends BaseActionViewItem {
 			this.dropdownMenu.show();
 		}
 	}
+
+	updateEnabled(): void {
+		if (this.getAction().enabled) {
+			if (this.actionItem) {
+				this.actionItem.classList.remove('disabled');
+			}
+
+			if (this.element) {
+				this.element.classList.remove('disabled');
+			}
+		} else {
+			if (this.actionItem) {
+				this.actionItem.classList.add('disabled');
+			}
+
+			if (this.element) {
+				this.element.classList.add('disabled');
+			}
+		}
+	}
```

Comment:
Maybe change to something more succinct?
```suggestion
updateEnabled(): void {
	const disabled = !this.getAction().enabled;
	this.actionItem?.classList.toggle('disabled', disabled);
	this.element?.classList.toggle('disabled', disabled);
}
```